README.md

```diff
@@ -14,12 +14,13 @@ In addition to full access to all the OpticStudio fucntions through the ZOS-API,
 - Wrapper functions for several OpticStudio analyses in `zospy.analyses`;
 - Easy access to solvers in `zospy.solvers`;
 - Easy access to all API constants in `zospy.constants`;
-- Autocomplete for all ZOS-API endpoints and constants.
+- Autocomplete for all ZOS-API endpoints and constants;
+- Automatic downcasting of common ZOS-API interfaces to their implementation.
```

Comment:
Do we want to mention this in the README? It requires a quite strong technical background to understand what this means, and I don't expect the average user to link this to Python.NET 3 related problems